lib/elixir/lib/keyword.ex

```diff
@@ -48,7 +48,7 @@ defmodule Keyword do
 
   For example, `Keyword.get/3` will get the first entry matching
   the given key, regardless if duplicated entries exist.
-  Similarly, `Keyword.put/3` and `Keyword.delete/3` ensure all
+  Similarly, `Keyword.put/3` ensures all
```

Comment:
Maybe refer to `Keyword.delete/2`?

```suggestion
  Similarly, `Keyword.put/3` and `Keyword.delete/2` ensure all
```